compiler/resolution/functionResolution.cpp

```diff
@@ -1770,8 +1962,13 @@ handleSymExprInExpandVarArgs(FnSymbol*  workingFn,
         // Otherwise, do the substitution now.
         subSymbol(workingFn->body, formal, var);
       }
+     } else {
+      // !needTupleInBody
+      substituteVarargTupleRefs(workingFn->body, n, formal, varargFormals);
+     }
 
```

Comment:
You've added a layer of conditional but failed to update the indentation on the existing body to match.